worlds/subnautica/__init__.py

```diff
@@ -1,3 +1,4 @@
+import functools
```

Comment:
This is unused by the code below?

Reply:
the decorator is @functools.cached_property

Reply:
oh its in init still.. yeah, that's where the Cache first existed before I moved it.

Reply:
> Hm, the .Cache is kind of awkward. Maybe call it `Definitions` or something and move the unsorted list in as well? Or you could use module-level getattr to provide the cached ones on module level, i think.

A module-level wrapper to cache on access could maybe be a generally available Utils class/type?